src/components/FilterList/SearchResults/SearchResultsPagination.tsx

```diff
@@ -0,0 +1,29 @@
+import { PaginationList } from '@/components/PaginationList/PaginationList';
+
+import { useSearch } from './hooks';
+import { SearchResultsProps } from '@/types/types';
```

Comment:
These imports need to be sorted

```suggestion
import { PaginationList } from '@/components/PaginationList/PaginationList';
import { SearchResultsProps } from '@/types/types';

import { useSearch } from './hooks';
```